app/elements/calendar-app.html

```diff
@@ -176,6 +180,13 @@
       renderColor: function(event, element) {
         var edition = (event.assignment ? event.assignment : event).course.edition;
 
+        var node = document.createElement("span");
+        var textnode = document.createTextNode(" [" + edition.courseId + "] ");
+        node.appendChild(textnode);
+        node.style.float = "left";
+        node.style.padding = "0 5px 0 0";
+        element[0].firstChild.children[1].appendChild(node);
```

Comment:
I fully agree it's ugly, but fullcalendar does not give a hook to add something to the calendar nodes (hence I'm inserting it in this method in the first place). They also don't have unique IDs, so this was the dirty fix for it